Dockerfile

```diff
@@ -14,8 +15,16 @@ RUN apk add --no-cache --virtual=.build-deps curl build-base postgresql-dev && \
     npm install --global yarn && \
     curl -o $ZOO_RDS_PRODUCTION_CERT_PATH https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem && \
     pip install --no-cache-dir -r requirements.txt -r test-requirements.txt && \
+    # gcloud setup
+    curl https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz > /tmp/google-cloud-sdk.tar.gz && \
+    mkdir -p /usr/local/gcloud && \
+    tar -C /usr/local/gcloud -xvf /tmp/google-cloud-sdk.tar.gz && \
+    /usr/local/gcloud/google-cloud-sdk/install.sh && \
     apk del .build-deps
 
+ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin
+RUN gcloud components install kubectl
```

Comment:
Same for `kubectl`. It might be easier to also use some client library, like [this one](https://github.com/kubernetes-client/python)